src/window.cpp

```diff
@@ -152,23 +160,35 @@ void window::loop(const window_events& ev)
 							ev.key_down(e.key.keysym.sym);
 					}
 					break;
-
 				case SDL_KEYUP:
 					if (ev.key_up)
 						ev.key_up(e.key.keysym.sym);
 					break;
+				case SDL_WINDOWEVENT:
+					if (e.window.event == SDL_WINDOWEVENT_RESIZED)
+					{
+						time_since_resize = clock::now();
+						back_buffers_	  = create_back_buffers(vec2u{ static_cast<unsigned int>(e.window.data1),
+																   static_cast<unsigned int>(e.window.data2) },
+															renderer_handle);
```

Comment:
The intent behind the `time_since_resize` logic was to delay this bit specifically - it's the recreation of the back-buffers every tick that causes the resize to be slow. Move this into the `if ((to_seconds(time - time_since_resize)) > 0.3 && window_resized)` down below.